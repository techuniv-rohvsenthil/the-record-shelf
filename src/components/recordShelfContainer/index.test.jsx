import React from 'react';
import { render } from '@testing-library/react';
import RecordShelfContainer from './index';


describe('the RecordShelfContainer component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<RecordShelfContainer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
