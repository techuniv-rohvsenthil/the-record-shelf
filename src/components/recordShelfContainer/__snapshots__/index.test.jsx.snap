// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the RecordShelfContainer component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <div>
      <div>
        The
      </div>
      <div>
        Record
      </div>
      <div>
        Shelf
      </div>
    </div>
    <button
      data-testid="sync-btn"
      type="button"
    >
      sync
    </button>
  </div>
</DocumentFragment>
`;
